src/Groups/matrices/MatGrp.jl

```diff
@@ -198,28 +198,32 @@ function assign_from_description(G::MatrixGroup)
    end
 end
 
+function _ring_iso(G::MatrixGroup{T}) where T
+  if !isdefined(G, :ring_iso)
+    if T === QQBarFieldElem
+      # get all matrix entries into one vector
+      entries = reduce(vcat, vec(collect(matrix(g))) for g in gens(G))
+      # construct a number field over which all matrices are already defined
+      nf, nf_to_QQBar = number_field(QQ, entries)
+      psi = pseudo_inv(nf_to_QQBar)
+      iso = iso_oscar_gap(nf)
+      G.ring_iso = MapFromFunc(G.ring, codomain(iso),
+                               x -> iso(preimage(nf_to_QQBar, x)),
+                               y -> preimage(iso,nf_to_QQBar(y))
```

Comment:
I know that this was there before, but shouldn't it be
```suggestion
                               y -> nf_to_QQBar(preimage(iso, y))
```
Also the map `psi` seems not to be used in this function.

Reply:
Ah indeed!

`psi` was from an earlier iteration of this code and I forgot to delete it.

I guess we should have a test case for this problem, too...